<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>agenti</class>
 <widget class="QDialog" name="agenti">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Anagarafica agenti</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="minimumSize">
          <size>
           <width>267</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>267</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Informazioni</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QTextEdit" name="dettagli">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_5">
            <property name="minimumSize">
             <size>
              <width>251</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>251</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="1" column="2">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Totale Agentii</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2" colspan="2">
              <widget class="QLineEdit" name="totale_list"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="2" column="7">
             <widget class="QPushButton" name="esci">
              <property name="text">
               <string>Esci</string>
              </property>
              <property name="icon">
               <iconset resource="../fabaria.qrc">
                <normaloff>:/images/dialog-close.png</normaloff>:/images/dialog-close.png</iconset>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="salva">
              <property name="text">
               <string>Salva</string>
              </property>
              <property name="icon">
               <iconset resource="../fabaria.qrc">
                <normaloff>:/images/document-save.png</normaloff>:/images/document-save.png</iconset>
              </property>
             </widget>
            </item>
            <item row="2" column="4">
             <widget class="QPushButton" name="elimina">
              <property name="text">
               <string>Elimina</string>
              </property>
              <property name="icon">
               <iconset resource="../fabaria.qrc">
                <normaloff>:/images/edit-delete.png</normaloff>:/images/edit-delete.png</iconset>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="nuovo">
              <property name="text">
               <string>Nuovo</string>
              </property>
              <property name="icon">
               <iconset resource="../fabaria.qrc">
                <normaloff>:/images/dialog-ok-apply.png</normaloff>:/images/dialog-ok-apply.png</iconset>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QPushButton" name="aggiorna">
              <property name="text">
               <string>Aggiorna</string>
              </property>
              <property name="icon">
               <iconset resource="../fabaria.qrc">
                <normaloff>:/images/view-refresh.png</normaloff>:/images/view-refresh.png</iconset>
              </property>
             </widget>
            </item>
            <item row="2" column="5">
             <widget class="QPushButton" name="stampa">
              <property name="text">
               <string>Stampa</string>
              </property>
              <property name="icon">
               <iconset resource="../fabaria.qrc">
                <normaloff>:/images/document-print.png</normaloff>:/images/document-print.png</iconset>
              </property>
             </widget>
            </item>
            <item row="2" column="6">
             <widget class="QPushButton" name="pdf">
              <property name="text">
               <string>Esporta pdf</string>
              </property>
              <property name="icon">
               <iconset resource="../fabaria.qrc">
                <normaloff>:/images/pdf.png</normaloff>:/images/pdf.png</iconset>
              </property>
             </widget>
            </item>
            <item row="2" column="8">
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="3">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Agenti</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>ID</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="id_agenti">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 0);</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Nome</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLineEdit" name="nome"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Cognome</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="cognome"/>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Indirizzo</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QLineEdit" name="indirizzo"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Località</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="localita"/>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>CAP</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="prov"/>
             </item>
             <item row="2" column="3">
              <widget class="QLineEdit" name="cap"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Prov.</string>
               </property>
              </widget>
             </item>
             <item row="3" column="3">
              <widget class="QLineEdit" name="telefono"/>
             </item>
             <item row="3" column="2">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Telefono</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1" colspan="3">
              <widget class="QLineEdit" name="email"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Email</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1" colspan="3">
              <widget class="QLineEdit" name="note"/>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Note</string>
               </property>
              </widget>
             </item>
             <item row="6" column="2" colspan="2">
              <widget class="QLineEdit" name="commissione"/>
             </item>
             <item row="7" column="1" colspan="4">
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="6" column="0" colspan="2">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Commissione  %</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Lista agenti</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QTableView" name="tableView">
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>21</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_10">
      <item row="0" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Cerca</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="cerca"/>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>334</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>id_agenti</tabstop>
  <tabstop>nome</tabstop>
  <tabstop>cognome</tabstop>
  <tabstop>indirizzo</tabstop>
  <tabstop>localita</tabstop>
  <tabstop>cap</tabstop>
  <tabstop>prov</tabstop>
  <tabstop>telefono</tabstop>
  <tabstop>email</tabstop>
  <tabstop>note</tabstop>
  <tabstop>commissione</tabstop>
  <tabstop>totale_list</tabstop>
  <tabstop>aggiorna</tabstop>
  <tabstop>dettagli</tabstop>
 </tabstops>
 <resources>
  <include location="../fabaria.qrc"/>
 </resources>
 <connections/>
</ui>
